src/mame/drivers/missb2.cpp

```diff
@@ -197,7 +191,7 @@ static ADDRESS_MAP_START( master_map, AS_PROGRAM, 8, missb2_state )
 	AM_RANGE(0xff98, 0xff98) AM_WRITENOP    // ???
 ADDRESS_MAP_END
 
-static ADDRESS_MAP_START( slave_map, AS_PROGRAM, 8, missb2_state )
+static ADDRESS_MAP_START( missb2_subcpu_map, AS_PROGRAM, 8, missb2_state )
```

Comment:
I originally did this with the intention to eventually merge missb2 and bublbobl.cpp into one driver, but I can undo it to reduce churn and unnecessary descriptiveness